python/openai/openai_frontend/engine/utils/triton.py

```diff
@@ -40,26 +40,30 @@ def _create_vllm_inference_request(
 
     # NOTE: The exclude_none is important, as internals may not support
     # values of NoneType at this time.
-    sampling_parameters = request.model_dump(
+    sampling_parameters = request.model_dump_json(
         exclude=excludes,
         exclude_none=True,
     )
-    inputs["text_input"] = [prompt]
-    inputs["stream"] = [request.stream]
     exclude_input_in_output = True
     echo = getattr(request, "echo", None)
     if echo:
```

Comment:
I believe this should be:
```
if echo is not None:
```
instead of ```if echo``` - because the next line assumes that echo could be `False`

Reply:
nice catch, done: https://github.com/triton-inference-server/server/pull/7682/commits/9f17b62ecec2ba6b2372d1f0252dcd7355075b86